src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.common.not-implemented.view :as not-implemented]
+            [quo2.components.settings.data-item.content.left-side :as left-side]
+            [quo2.components.settings.data-item.content.right-side :as right-side]))
+
+(def view-internal
+  (fn [{:keys [blur? card? icon-right? label description status size theme on-press title subtitle
+               icon]}]
+    (let [icon-color (cond
+                       (or blur? (= :dark theme)) colors/white
+                       (= :light theme)           colors/neutral-100)]
```

Comment:
For this one, I think it's the same as:
```clojure
(cond
  (or blur? (= :dark theme)) colors/white
  :else                      colors/neutral-100)
```

which is the same as an `if` (and easier to unserstand IMO):
```clojure
(if (or blur? (= :dark theme))
  colors/white
  colors/neutral-100)
```

Reply:
Done